File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char,int> mps;
        unordered_map<char,int> mpt;
        for( char ch : s){
            mps[ch]++;
        }
        for(char ch : t){
            mpt[ch]++;
        }
        return mps == mpt;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        maps={}
        mapt={}
        for ch in s:
            if ch in maps.keys():
                maps[ch]+=1
            else:
                maps[ch]=1
        
        for ch in t:
            if ch in mapt.keys():
                mapt[ch]+=1
            else:
                mapt[ch]=1
        return maps == mapt